.github/workflows/wheels.yml

```diff
@@ -51,9 +51,14 @@ jobs:
         # https://github.com/github/feedback/discussions/7835#discussioncomment-1769026
         buildplat:
         - [ubuntu-20.04, manylinux_x86_64]
+        # TODO: Consider re-enabling macos wheels, once meson-python makes it easy to
+        # cross compile
         - [macos-11, macosx_*]
+        #- [macos-11, macosx_x86_64]

```

Comment:
Do these macos and windows wheels still need disabling?

Reply:
Not anymore for macOS.
Windows 32-bit could probably be worked out, but given the low percentage of downloads(~1% of users use them), I am dropping the wheel builds.

(We would still test on 32-bit, but users would just have to compile themselves).